components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx

```diff
@@ -290,24 +279,13 @@ export const paymentFailedBanner = () => {
                 />
             }
             message={
-                <>
-                    <FormattedMessage
-                        id='billing.subscription.info.mostRecentPaymentFailed.description.mostRecentPaymentFailed'
-                        defaultMessage='It looks your most recent payment failed because the credit card on your account has expired. Please '
-                    />
-                    <BlockableLink
-                        to='/admin_console/billing/payment_info'
-                    >
-                        <FormattedMessage
-                            id='billing.subscription.info.mostRecentPaymentFailed.description.updatePaymentInformation'
-                            defaultMessage='update your payment information'
-                        />
-                    </BlockableLink>
-                    <FormattedMessage
-                        id='billing.subscription.info.mostRecentPaymentFailed.description.avoidAnyDisruption'
-                        defaultMessage=' to avoid any disruption.'
-                    />
-                </>
+                <FormattedMessage
+                    id='billing.subscription.info.mostRecentPaymentFailed.description.mostRecentPaymentFailed'
+                    defaultMessage='It looks your most recent payment failed because the credit card on your account has expired. Please  <link>update your payment information</link> to avoid any disruption.'
```

Comment:
Neat! Didn't know you could specify tags to wrap content with arbitrary JSX elements like this.

Reply:
me neither, special thanks to @hmhealey for that advice: @neallred this is the reference link https://formatjs.io/docs/react-intl/components/#rich-text-formatting

Reply:
Yeah, it's a great feature. It's sort of like the older way we did things where we'd inline HTML, but it's easier to use since it lets us use React components and hide the HTML while also being safer since it prevents potentially malicious translators from breaking anything.